src/contexts/pageContext.tsx

```diff
@@ -15,17 +15,15 @@ const defaultFlashProps: FlashProps = {
 
 const PageContext = createContext<PageContextType>({
   flashProps: defaultFlashProps,
-  setFlashProps(this: PageContextType, props: FlashProps) {
-    this.flashProps = props
-  },
+  setFlashProps: () => {},
```

Comment:
This function was causing test failures in CI (but, weirdly, not in dev). Since the default context value should never be used anyway, it made sense to just make it a noop.